ui/src/shared/utils/checkQueryResult.ts

```diff
@@ -13,7 +13,7 @@
 
   See https://github.com/influxdata/flux/blob/master/docs/SPEC.md#errors.
 */
-export const checkQueryResult = (file: string): void => {
+export const checkQueryResult = (file: string = ''): void => {
```

Comment:
what does this do?

Reply:
**checkQueryResult** is called here https://github.com/influxdata/influxdb/blob/b653c2f975ae6be20e99b580e66058fb7af5a788/ui/src/shared/components/TimeSeries.tsx#L231

At runtime, `result.csv` may be undefined (not a string) when there is an error. The code immediately above the call to **checkQueryResult** throws a runtime error which will hide the fact that **checkQueryResult** will throw its own error when `result.csv` is not a string. By adding a default value to the parameter we prevent checkQueryResult from throwing its own error (which would be visible in the console should the throws above it are ever removed).

Reply:
Specifically, the error messaging will become checkQueryResult's thrown error rather than the actual error. And we want the error message to be useful to the user (rather than the dev). The user won't care or know what to do with "Cannot read property 'slice' of undefined" means.

<img width="1680" alt="Screen Shot 2020-04-15 at 11 43 58 AM" src="https://user-images.githubusercontent.com/10736577/79376099-82443800-7f0e-11ea-9e8d-1257497124c0.png">